### DISCUSSION TITLE
Trying to turn on prometheus metrics for KEDA

### DISCUSSION THREAD

**Discussion by swails** [kedacore/charts#172] [Q&A]
I'm currently trying to turn on as many Prometheus metrics as I can from Keda so we can ingest these time series to get as full of a picture as we can about what Keda and its spawned resources are doing.  I've successfully set `prometheus.metricsServer.enabled` to `true` and I see those metrics coming in.

I see mention of `prometheus.metricServer.podMonitor.enabled` without much documentation, so I enabled that and got the following message (in a helm deploy via the terraform-helm provider):

```
Error: unable to recognize "": no matches for kind "PodMonitor" in version "monitoring.coreos.com/v1"
```

I'd like to understand better what the podMonitor metrics are, what their relationship is to k8s and keda, what metrics those actually make available, and what I should expect to see if this functionality is "working."

**Comment by tomkerkhove** [comment]
PodMonitor is a way in Prometheus to define what needs to be scraped, learn more on https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md#include-podmonitors.

You are getting that exception because the Prometheus Operator is not installed or it is missing one of the CRDs for it.
